<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fenetre</class>
 <widget class="QWidget" name="fenetre">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Correcteur du scan de MG2550S</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="grpInput">
     <property name="title">
      <string>Entrée</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="lFileIn">
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>lien de l'image</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bFileIn">
          <property name="text">
           <string>Parcourir</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="bShow">
        <property name="text">
         <string>Appercu</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkReplaceImage">
        <property name="text">
         <string>Remplacer l'image actuelle</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Notes:caractéristiques de l'image requises :
    résolution de 300ppp
    format png
    taille de 2550x3507 pixels</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QGroupBox" name="grpOutput">
     <property name="title">
      <string>Sortie</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="bCorriger">
        <property name="text">
         <string>Corriger l'image</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="lblFileOut">
          <property name="text">
           <string>Enregistré avec le nom</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lFileOut">
          <property name="text">
           <string/>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QProgressBar" name="pbPixels">
        <property name="value">
         <number>0</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignJustify|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
